paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAccountManager.kt

```diff
@@ -291,6 +279,23 @@ internal class DefaultLinkAccountManager @Inject constructor(
         }
     }
 
+    override suspend fun shareCardPaymentDetails(
+        paymentDetailsId: String,
+        paymentMethodCreateParams: PaymentMethodCreateParams
+    ): Result<LinkPaymentDetails.Saved> {
+        return runCatching {
+            requireNotNull(linkAccountHolder.linkAccountInfo.value.account)
+        }.mapCatching { account ->
+            linkRepository.shareCardPaymentDetails(
+                id = paymentDetailsId,
+                last4 = paymentMethodCreateParams.cardLast4().orEmpty(),
+                consumerSessionClientSecret = account.clientSecret,
+                paymentMethodCreateParams = paymentMethodCreateParams,
+                allowRedisplay = paymentMethodCreateParams.allowRedisplay,
```

Comment:
Not introduced in this pull request, but we can remove this parameter since we already pass the entire `paymentMethodCreateParams`. Same for `last4`.